library/src/main/java/com/rengwuxian/materialedittext/MaterialEditText.java

```diff
@@ -331,6 +331,14 @@ public MaterialEditText(Context context, AttributeSet attrs, int style) {
     init(context, attrs);
   }
 
+  public String getString() {
+    return getText().toString().trim();
```

Comment:
I would not put it at all (my point of view). this is an edittext, it should not care about how you want to handle the string typed